templates/default/dump_audit_logs_to_hdfs.sh.erb

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env bash
+
+LOGS_DIR=<%= @weblogs_dir %>
+HADOOP_HOME=<%= node['hops']['base_dir'] %>
+HDFS_WEBLOGS_DIR=<%= @remote_weblogs_dir %>
+
+CHECKPOINT=$LOGS_DIR/.checkpoint
+FILES_TO_DUMP=$LOGS_DIR/.files_to_dump
+REMOTE_FILE_PERMISSIONS=0750
+# This will match only files in the format of server_audit_log0.log
+REGEX=".*server_audit_log.*.log"
```

Comment:
How about switching the general 'any' character with only numbers?

Reply:
It can be with out any number too